python/mleap/sklearn/feature_extraction/text.py

```diff
@@ -65,4 +105,74 @@ def serialize_to_bundle(self, transformer, path, model_name):
                   "port": "output"
                    }]
 
-        self.serialize(transformer, path, model_name, attributes, inputs, outputs)
\ No newline at end of file
+        self.serialize(transformer, path, model_name, attributes, inputs, outputs)
+
+
+class TfidfVectorizerSerializer(MLeapSerializer):
+    pipeline_serializer = PipelineSerializer()
+
+    def __init__(self):
+        super(TfidfVectorizerSerializer, self).__init__()
+
+    @staticmethod
+    def set_prediction_column(transformer, prediction_column):
+        transformer.prediction_column = prediction_column
+
+    @staticmethod
+    def set_input_features(transformer, input_features):
+        transformer.input_features = input_features
+
+    def serialize_to_bundle(self, transformer, path, model_name):
+        num_features = transformer.idf_.shape[0]
+        vocabulary = [None] * num_features
+
+        for term, termidx in transformer.vocabulary_.iteritems():
+            vocabulary[termidx] = str(term)
+
+        tf_attributes = [
+            ('vocabulary', vocabulary),
+            ('binary', transformer.binary),
+            # no concept of 'min_tf' in scikit's TfidfVectorizer
+            ('min_tf', 0)
+        ]
+        tf_inputs = [{
+            "name": transformer.input_features,
+            "port": "input"
+        }]
+        tf_outputs = [{
+            "name": "token_counts".format(transformer.name),
+            "port": "output"
+        }]
+        tf_step = TfidfStep(transformer, 'count_vectorizer', tf_attributes, tf_inputs, tf_outputs)
+
+        idf_attributes = [
+            ('idf', transformer.idf_)
```

Comment:
I think it needs to be transformer.idf_.tolist() for the serialized model to work correctly with mleap serving.